api/commands/format.ts

```diff
@@ -13,7 +13,7 @@ const JS_BEAUTIFY_OPTIONS: JsBeautifyOptions = {
 
 const MAX_SQL_FORMAT_ATTEMPTS = 5;
 
-const TEXT_LIFT_PATTERNS = [/r'(\\['\\]|[^\n'\\])*'/, /r"(\\["\\]|[^\n"\\])*"/];
+const TEXT_LIFT_PATTERNS = [/r'.*'/, /r".*"/];
```

Comment:
i am a bit worried that these are very greedy. e.g. what happens with `select r'foo', 'bar'`? i'm betting it lifts all of `r'foo', 'bar'` into a single string.

Reply:
Ah good spot. I updated regex to not be greedy, which then broke the parsing of escaped quotes, so I added a negative lookahead to fix that.